web_src/js/features/repo-common.js

```diff
@@ -1,6 +1,10 @@
 import $ from 'jquery';
```

Comment:
Also, I recommend we move all this code to `web_src/js/features/citation.js` so it's cleanly separated.